tests/run-test.py

```
@@ -282,19 +321,19 @@ def on_line(self, line, line_number):
                 self.is_mysql and not MySQLCompare.matched(self.outputs, self.matches))):
                 return False
             self.query_line_number = line_number
             self.is_mysql = True
             self.query = line[len(CURL_TIDB_STATUS_PREFIX):]
             self.outputs, err = self.executor_curl_tidb.exe(self.query)
-            if err != None:
+            if err is not None:
                 return False
             self.matches = []
         elif line.startswith(CMD_PREFIX) or line.startswith(CMD_PREFIX_ALTER):
             if verbose: print(f'{datetime.datetime.now().strftime("%H:%M:%S.%f")} running {line}')
```

Ruff (0.15.14)

[error] Multiple statements on one line (colon)

(E701)